.golangci.yaml

```diff
@@ -0,0 +1,26 @@
+run:
+  timeout: 1m
+  tests: true
+
+linters:
+  disable-all: true
```

Comment:
Instead of disabling all linters and then enabling some, can you instead leave the default linters in place and only disable those which currently produce too many warnings?

Reply:
This was a copy-paste from https://github.com/go-logr/logr/blob/master/.golangci.yaml, apart from two commented out linters that did produce many warnings. 
The disabled linters are not default ones, default linters do not produce any findings at all.
I removed `disable-all` config line, removed default linters from enabled-ones, and added godot. We'll need one more code-changing PR before this can be merged to fix comments to end with a dot.